src/Atmos/Dycore/test/runtests.jl

```diff
@@ -0,0 +1,28 @@
+using MPI, Test
+
+@testset "Atmos/Dycore" begin
+  # The code below was modified from the MPI.jl file runtests.jl
+  #
+  # Code coverage command line options; must correspond to src/julia.h
+  # and src/ui/repl.c
+  JL_LOG_NONE = 0
+  JL_LOG_USER = 1
+  JL_LOG_ALL = 2
+  coverage_opts = Dict{Int, String}(JL_LOG_NONE => "none",
+                                    JL_LOG_USER => "user",
+                                    JL_LOG_ALL => "all")
+  coverage_opt = coverage_opts[Base.JLOptions().code_coverage]
+  testdir = dirname(@__FILE__)
+
+  for (n, f) in [(1, "../drivers/isentropic_vortex.jl")
```

Comment:
should we move them into the `test` directory?

Reply:
I'm fine with that. Will do so.

Reply:
resolved by a91dbbc